gulp/tasks/views.js

```diff
@@ -11,11 +11,12 @@ gulp.task('views', function() {
   gulp.src('app/index.html')
     .pipe(gulp.dest(config.dist.root));
 
-  // Process any other view files from app/views
+  // Process any other view files from app/modules
   return gulp.src(config.views.src)
-    .pipe(templateCache({
-      standalone: true
+    .pipe(templateCache('templates.templates.js', {
+      standalone: false,
+      module: 'app.templates'
```

Comment:
I think we should be putting the templates in their parent module, rather than in the global module.

https://github.com/perry/old-weather/blob/master/gulpfile.js#L122-L136